day2_exercises/fMRI_homework_20151107.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Nov  7 09:02:19 2015
+
+@author: Elissaios
+"""
+
+#%% Load nii image (after it was unzipped)
+import nibabel as nib
+
+folder = r"c:\Anaconda3\Lib\site-packages\nibabel\tests\data";
```

Comment:
Try very hard to avoid hard-coding paths like this. I guess you are debugging, to use the nibabel test image instead of the openfmri image?